src/lib/calculator.ts

```diff
@@ -0,0 +1,79 @@
+import { PLUS, SUBTRACT, MULTIPLY, DIVIDE, OPERATORS_REGEX, INIT_OPERAND } from '@/constants'
+
+type Operator = typeof PLUS | typeof SUBTRACT | typeof MULTIPLY | typeof DIVIDE
+
+export default class Calculator {
+  operands: number[]
+  operators: Operator[]
```

Comment:
계산기는 항상 마지막에 입력된 연산자 하나만 관심을 두도록 만드는 것이 현재 요구사항에 더 적절할 것 같아요. 따라서 다음과 같은 타입이 계산기에서 연산자가 가져야할 상태를 적절히 표현할 것이라고 생각됩니다. 😄 
```suggestion
  operator: Operator | null
```